data/scripts/creaturescripts/player/update_client_on_advance.lua

```diff
@@ -0,0 +1,17 @@
+local updateClientOnAdvance = CreatureEvent("Update Client On Advance")
+
+function updateClientOnAdvance.onAdvance(player, skill, oldLevel, newLevel)
```

Comment:
Not at all. This script is not doing "exactly what the name say" because then the script name should be named "update_client_exp_display_on_level_change_and_take_screenshot_on_any_skill_change.lua"

Reply:
IMO, no reason to remove the other two files.

Reply:
This new script has way less semantic and not a single reponsibility.

Reply:
And please, do not resolve my comments if there is a discussion on going.